### PR TITLE
on bulk upsert set record count to number of objects in request

### PR DESCRIPTION
**Pull request by annyhe**

### CODE REVIEW

**Comment by annyhe** [comment]
screen shot 2017-02-03 at 10 53 43 am

File: utils/apiLog.js [hunk 1]
```diff
@@ -31,8 +31,9 @@ function getSize(obj) {
  * @param {Object} resultObj - API result object
  * @param {Object} logObject - from the request object
  * @param {Object or Array} retval - the returned object
+ * @param {Integer} customRecordCount - a custom recordCount
```
Comment (iamigo) [inline utils/apiLog.js]: pls. rename this to "recordCountOverride" and call it out as optional in the field description in the @param comments

File: utils/apiLog.js [hunk 2]
```diff
@@ -68,10 +73,11 @@ function mapApiResultsToLogObject(resultObj, logObject, retval) {
  * @param {Object} resultObj - API result object
  * @param {Object} logObject - from the request object
  * @param {Object or Array} retval - the returned object
+ * @param {Integer} recordCount - a custom recordCount
```
Comment (iamigo) [inline utils/apiLog.js]: pls. rename this to "recordCountOverride" and call it out as optional in the field description in the @param comments

File: utils/apiLog.js [hunk 3]
```diff
@@ -81,8 +87,9 @@ function combineAndLog(resultObj, logObject, retval) {
  * @param {Object} req - the request object
  * @param {Object} resultObj - Object with the rest of the fields to print
  * @param {Object or Array} retval - the returned object
+ * @param {Integer} recordCount - a custom recordCount
```
Comment (iamigo) [inline utils/apiLog.js]: pls. rename this to "recordCountOverride" and call it out as optional in the field description in the @param comments

**Pull request merged**